lib/compat/wordpress-6.2/class-gutenberg-rest-block-pattern-categories-controller.php

```diff
@@ -29,7 +29,7 @@ public function prepare_item_for_response( $item, $request ) {
 		$data   = array();
 		foreach ( $keys as $key ) {
 			if ( rest_is_field_included( $key, $fields ) ) {
-				$data[ $key ] = $item[ $key ];
+				$data[ $key ] = isset( $item[ $key ] ) ? $item[ $key ] : '';
```

Comment:
Good catch, should the fallback be an empty string or null?

Reply:
The schema says string, that’s the reason I decided on empty string fallback.

Reply:
Could we make it optional and not set it at all maybe? I feel empty string is misleading.

Reply:
Sounds good.

Reply:
It is updated in [548fba5](https://github.com/WordPress/gutenberg/pull/45410/commits/548fba51f0c30edf9a92741169cd523ecf9b05cd). 

Now it matches behavior for patterns endpoint - https://github.com/WordPress/wordpress-develop/blob/trunk/src/wp-includes/rest-api/endpoints/class-wp-rest-block-patterns-controller.php#L134-L138.